vc_assistance/indico_vc_assistance/util.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from sqlalchemy.orm import joinedload
+
+from indico.core.db.sqlalchemy.util.queries import limit_groups
+from indico.modules.events import Event
+from indico.modules.events.requests.models.requests import Request, RequestState
+from indico.modules.vc import VCRoomEventAssociation
+
+
+def is_vc_manager(user):
+    """Checks if a user is a VC manager"""
+    return _is_acl(user, 'managers')
+
+
+def is_vc_support(user):
+    """Checks if a user is a VC support"""
+    return _is_acl(user, 'vc_support')
+
+
+def _is_acl(user, acl):
+    from indico_vc_assistance.plugin import VCAssistanceRequestPlugin
+    if user.is_admin:
+        return True
+    return VCAssistanceRequestPlugin.settings.acls.contains_user(acl, user)
+
+
+def has_room_with_vc_attached(event):
+    return any(vc for vc in VCRoomEventAssociation.find_for_event(event, include_hidden=True)
+               if vc.link_object.room.has_vc)
```

Comment:
Won't this fail if the linked object has no room associated?

Reply:
`has_vc` is legacy and will go away. see `get_av_capable_rooms` on how to check for it now (we'll have a "Video conference" room feature)

Reply:
Does it mean that we need a `room_feature` field on the plugin settings as we have for audiovisual?

Reply:
yes